apis/compute/v1beta1/firewallpolicyrule_identity.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/GoogleCloudPlatform/k8s-config-connector/apis/common"
+	refsv1beta1 "github.com/GoogleCloudPlatform/k8s-config-connector/apis/refs/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type FirewallPolicyRuleIdentity struct {
+	id     int64
+	parent *FirewallPolicyRuleParent
+}
+
+func (i *FirewallPolicyRuleIdentity) String() string {
+	p := strconv.Itoa(int(i.id))
+	return i.parent.String() + "/rules/" + p
+}
+
+func (i *FirewallPolicyRuleIdentity) Parent() *FirewallPolicyRuleParent {
+	return i.parent
+}
+
+func (i *FirewallPolicyRuleIdentity) ID() int64 {
+	return i.id
+}
+
+type FirewallPolicyRuleParent struct {
+	FirewallPolicy string
+}
+
+func (p *FirewallPolicyRuleParent) String() string {
+	return "locations/global/firewallPolicies/" + p.FirewallPolicy
```

Comment:
Everytime  I saw this I feel like something's wrong. So just to double check, the correct URL is "locations/global"?